src/components/Article/index.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { PrismicRichText } from "@prismicio/react";
+import { H1, P, MarginContainer } from "../../styles/styles";
+import { ArticleContainer, Section, StyledP, StyledImage } from "./styled";
+
+const Article = ({
+  title,
+  content,
+  author,
+  imageUrl,
+  imageAlt,
+  imageCaption,
+}) => {
+  return (
+    <MarginContainer>
+      <ArticleContainer>
+        <Section>
+          <H1>{title}</H1>
+          <P>{author}</P>
+        </Section>
+        <Section>
+          <StyledImage src={imageUrl} alt={imageAlt} />
+          <P>{imageCaption} </P>
+        </Section>
+        <Section>
+          <StyledP>
+            <PrismicRichText field={content} />
+          </StyledP>
```

Comment:
`<PrismicRichText />` has a `components` prop that allows you to control the output of different elements in rich text, see docs [here](https://prismic.io/docs/technologies/template-fields-gatsby#rich-text-and-titles) (and there's an example in the [video page pr](https://github.com/Scout-NU/scout-presents-site/pull/14/files#diff-3f8a5baf91118745fc1b50041ac882a0ce473b5aab68c615832d9aad05a0ac5bR54-R59))
that way you don't have to wrap the elements in a `<p>` and we can also use our custom `<H2>` and `<H3>` (it only works on block elements) with rich text!!